src/c++/perf_analyzer/schema.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/examples/schema.json",
+    "title": "Perf Analyzer output data",
+    "description": "A json file describing the output from a perf analyzer run.",
+    "type": "object",
+    "required": [
+        "experiments",
+        "version"
+    ],
+    "properties": {
+        "experiments": {
+            "description": "The array of all experiments run by perf analyzer.",
+            "type": "array",
+            "required": [
+                "experiment",
+                "requests",
+                "window_boundaries"
+            ],
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "experiment": {
+                        "description": "A single experiment run by perf analyzer.",
+                        "type": "object",
+                        "required": [
+                            "mode",
+                            "value"
+                        ],
+                        "minItems": 1,
+                        "maxItems": 1,
+                        "properties": {
+                            "mode": {
+                                "description": "Operating mode of perf analyzer: For example, concurrency or request rate.",
+                                "type": "string"
+                            },
+                            "value": {
+                                "description": "Concurrency or request rate for the current experiment.",
+                                "type": "number"
+                            }
+                        }
+                    },
+                    "requests": {
+                        "description": "The array of requests sent by perf_analyzer for this experiment.",
+                        "type": "array",
+                        "uniqueItems": true,
+                        "items": {
+                            "$ref": "#/properties/experiments/items/properties/$defs/request"
+                        }
+                    },
+                    "$defs": {
+                        "request": {
+                            "description": "Info for a single request.",
+                            "type": "object",
+                            "required": [
+                                "timestamp",
+                                "responses_timestamps"
+                            ],
+                            "properties": {
+                                "timestamp": {
+                                    "description": "Time stamp of the request.",
+                                    "type": "number"
```

Comment:
Should [`integer`](https://json-schema.org/understanding-json-schema/reference/numeric.html#integer) be used for a lot of these since we know all timestamps are integers (no decimal)?

Reply:
It is a slightly tighter constraint. it should not matter in the end, but it locks the behavior down slightly more. Updated

Reply:
I think it's fine to be have a more accurate constraint here since the current version of PA only allows for integer timestamps. And if that changes, we have a versioning component of this schema to handle that.

Reply:
works for me